apps/backend/src/eventHandlers/MailService/SMTPMailService.ts

```diff
@@ -141,21 +141,30 @@ export class SMTPMailService extends MailService {
     });
   }
 
-  // TODO: This might need some attention from STFC and return the templates used in their email sending service.
   async getEmailTemplates(
     includeDraft = false
   ): ResultsPromise<STFCEmailTemplate[]> {
     return {
       results: [
         {
-          id: 'my-first-email',
-          name: 'My First Email',
-          description: 'A test message from STFC',
+          id: 'call-created-email',
+          name: 'Call Created Email',
         },
         {
-          id: 'my-second-email',
-          name: 'My Second Email',
-          description: 'A test message from STFC',
+          id: 'clf-proposal-submitted-pi',
```

Comment:
I see that you hardcode the templates here. Maybe this is temporary or you want to keep it like this for now if you cannot fetch them from an endpoint.